server.py

```diff
@@ -38,6 +57,29 @@ def add_note():
 
     return jsonify(response)
 
+@app.route('/notes/ascend')
+def ascend_order():
+    """ Returns notes in Descending Order """
+
+    notes = gather_all_notes_from_db()
+    
+    result = users_schema.dump(notes)
+    print "RRRRRRRRRRRR", result.data 
+
+    return jsonify(result.data)
+    
+
+
+@app.route('/notes/desc')
+def descend_order():
+    """ Returns notes in Descending Order """
+    notes =  Note.query.order_by(asc(Note.created_at)).all()
+
+    result = users_schema.dump(notes)
+    print "OOOOOOOOOOOO", result.data
+
+    return jsonify(result.data)
```

Comment:
Try combining `ascend_order()` and `descend_order()` into one function.  Lose the `print`s.